models/Item.yaml

```diff
@@ -0,0 +1,132 @@
+type: object
```

Comment:
@TomasBankauskas Can you please review the model, I'm not really up to date with the latest model feature set.

Should it be called `ItemBlock`? I see we now have models called `Link.yaml` and `Social.yaml` so either we are no longer following this naming convention, or this model is not a Block.

If we are no longer following the naming convention, we need to remove Block from all the old models.

If we are still following the naming convention, can we get a clear definition of when to name something with `thingBlock` or `thing`